crates/optimism/rpc/src/lib.rs

```diff
@@ -26,3 +26,12 @@ pub struct OptimismApi<Provider, Pool, Network, EvmConfig> {
     /// All nested fields bundled together.
     inner: Arc<EthApiInner<Provider, Pool, Network, EvmConfig>>,
 }
+
+unsafe impl<Provider, Pool, Network, EvmConfig> Send
+    for OptimismApi<Provider, Pool, Network, EvmConfig>
+{
+}
+unsafe impl<Provider, Pool, Network, EvmConfig> Sync
+    for OptimismApi<Provider, Pool, Network, EvmConfig>
+{
+}
```

Comment:
this shouldn't be necessary I think

Reply:
yes I know...though it seems, as long as `OptimismApi` isn't used anywhere the compiler doesn't do the job of inferring the `Send` and `Sync` impl . strangely enough I needed the manual impls to compile.